src/test/scala/com/crealytics/spark/v2/excel/MaxRowsReadSuite.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2022 Martin Mauch (@nightscape)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.crealytics.spark.v2.excel
+
+import com.holdenkarau.spark.testing.DataFrameSuiteBase
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+class MaxRowsReadSuite extends AnyWordSpec with DataFrameSuiteBase with Matchers {
+
+  "excel v2 and maxNumRows" can {
+
+    "read with maxNumRows=200" in {
+
+      val dfExcel = spark.read
+        .format("excel")
+        .option("path", "src/test/resources/v2readwritetest/large_excel/largefile-wide-single-sheet.xlsx")
+        .option("header", value = false)
```

Comment:
should we test with inferSchema= true and header = true too to get greater coverage?

Reply:
if you liek that, you will need the new xlsx too.. (has headers)